models/mlflow/README.md

```diff
@@ -34,6 +34,18 @@ Example:
 mkdir -p /opt/triton_models
 ```
 
+## Download Morpheus reference models
+
+The Morpheus reference models can be found in the [Morpheus](https://github.com/NVIDIA/Morpheus) repo. A script is provided to fetch the models using git-lfs due to size.
+Before running the MLflow plugin container, you can fetch the models and mount them to the local path on your host (for example, `/opt/triton_models`).
+
+```bash
+git clone https://github.com/NVIDIA/Morpheus.git morpheus
+cd morpheus
+scripts/fetch_data.py fetch models
+cp -RL models /opt/triton_models
```

Comment:
Shouldn't `/opt/triton_models` be empty initially because that will act as the Triton model repo directory when we start Triton in the next step. 

Also, doesn't appear that the models will be accessible to container when we call `publish_model_to_mlflow.py` unless we do a volume mount (-v) in the `docker run`.

Reply:
Maybe update code block to:
```
python publish_model_to_mlflow.py \
	--model_name sid-minibert-onnx \
	--model_directory /triton_models/models/triton-model-repo/sid-minibert-onnx \
    --flavor triton
```